### PR TITLE
[Run 291, 296, 320, 321]  Made the SettingsVC and AppInfoVC screen as in figma

### PR DESCRIPTION
**Pull request by arti1233**
Description  📝

Made the settings screen and the about app screen as in figma. Added clear navigation controller, title as in figma, added scroll on descriptions in AppInfoVC. 

---

Tests ✅

Simulator: iPhone SE gen 2 , 11, 14 Pro Max
Device : XR, iPhone 7

### CODE REVIEW

**Review by Qeezii — APPROVED** [review]

**Comment by Kengii** [comment]
В фигме цвет у title navigation в settings другой
UIColor(red: 0.825, green: 0.77, blue: 0.677, alpha: 1)

**Comment by Kengii** [comment]
iphone se 1gen not showing word back in navbar
Figma:
Снимок экрана 2023-02-01 в 22 07 12
Device:
Снимок экрана 2023-02-01 в 22 06 39

File: Runar/Runar/Controllers/Misc/TabBarController/MainTabBarController.swift [hunk 1]
```diff
@@ -50,6 +50,7 @@ class MainTabBarController: UITabBarController {
         let navigationVC = UINavigationController(rootViewController: rootViewController)
         navigationVC.tabBarItem.image = image
         navigationVC.tabBarItem.title = title
+        navigationVC.navigationBar.setBackgroundImage(UIImage(), for: .default)
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Controllers/Misc/TabBarController/MainTabBarController.swift:53]: Why do you use simple UIImage?

**Review by AliaksandrNiehaj — CHANGES_REQUESTED** [review]

File: Runar/Runar/Controllers/Settings flow/AppInfoVC.swift [hunk 2]
```diff
@@ -20,35 +21,29 @@ class AppInfoVC: UIViewController, UITextViewDelegate {
...
         textView.textContainer.lineBreakMode = .byWordWrapping
+        textView.showsVerticalScrollIndicator = false
         
         let string = String.descriptionAppText
         let firstUrl = "https://lyod1.bandcamp.com/releases"
         let secUrl = "https://danheimmusic.com"
  
         let paragraphStyle = NSMutableParagraphStyle()
-        paragraphStyle.lineHeightMultiple = 1.08
+        paragraphStyle.lineHeightMultiple = 1.17
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Controllers/Settings flow/AppInfoVC.swift]: Lets use constants.

File: Runar/Runar/Controllers/Settings flow/AppInfoVC.swift [hunk 3]
```diff
@@ -20,35 +21,29 @@ class AppInfoVC: UIViewController, UITextViewDelegate {
...
         textView.translatesAutoresizingMaskIntoConstraints = false
         let firstRange = (string as NSString).range(of: firstUrl)
         let secRange = (string as NSString).range(of: secUrl)
         
-        var attributedText = NSMutableAttributedString(string: string, attributes: [NSMutableAttributedString.Key.paragraphStyle: paragraphStyle,NSMutableAttributedString.Key.font: FontFamily.SFProDisplay.regular.font(size: 17), NSMutableAttributedString.Key.foregroundColor: UIColor.settingsWhiteText])
+        var attributedText = NSMutableAttributedString(string: string, attributes: [NSMutableAttributedString.Key.paragraphStyle: paragraphStyle,NSMutableAttributedString.Key.font: FontFamily.SFProDisplay.light.font(size: 20), NSMutableAttributedString.Key.foregroundColor: UIColor.settingsWhiteText])
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Controllers/Settings flow/AppInfoVC.swift]: Lets create constant for Font and it later.

File: Runar/Runar/Controllers/Settings flow/AppInfoVC.swift [hunk 4]
```diff
@@ -61,54 +56,34 @@ class AppInfoVC: UIViewController, UITextViewDelegate {
...
-            scrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
-            scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
-            
-            descriptionView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: descriptionTopOffset),
-            descriptionView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant:  24),
-            descriptionView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant:  -24),
-            descriptionView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
-        ])
+        descriptionView.snp.makeConstraints {
+            $0.leading.trailing.equalToSuperview().inset(24)
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Controllers/Settings flow/AppInfoVC.swift]: Use constants, please.

File: Runar/Runar/Controllers/Settings flow/SettingsVC.swift [hunk 5]
```diff
@@ -59,36 +58,17 @@ final class SettingsVC: UIViewController {
...
-            label.textColor = .settingsTitleColor
-            label.font = FontFamily.SFProDisplay.medium.font(size: 20)
-            label.textAlignment = .left
-            self.navigationItem.leftBarButtonItem = UIBarButtonItem(customView: label)
-        }
-
-        navigationController?.navigationBar.backgroundColor = .navBarBackground
-        navigationController?.setStatusBar(backgroundColor: .navBarBackground)
+        navigationController?.navigationBar.backgroundColor = .clear
+        navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.font: FontFamily.AmaticSC.bold.font(size: 36), NSAttributedString.Key.foregroundColor: UIColor(red: 0.825, green: 0.77, blue: 0.677, alpha: 1)]
```
Comment (AliaksandrNiehaj) [inline Runar/Runar/Controllers/Settings flow/SettingsVC.swift]: Need to use constant for color

**Review by AliaksandrNiehaj — CHANGES_REQUESTED** [review]

File: Runar/Runar/Controllers/Misc/TabBarController/MainTabBarController.swift (hunk 1 above)
Comment (AliaksandrNiehaj) [reply Runar/Runar/Controllers/Misc/TabBarController/MainTabBarController.swift:53]: @arti1233 You still don't answered this question.

File: Runar/Runar/Controllers/Settings flow/AppInfoVC.swift (hunk 3 above)
Comment (AliaksandrNiehaj) [reply Runar/Runar/Controllers/Settings flow/AppInfoVC.swift]: Moreover, use, please, short key values for attributed string.

File: Runar/Runar/Controllers/Misc/TabBarController/MainTabBarController.swift (hunk 1 above)
Comment (arti1233) [reply Runar/Runar/Controllers/Misc/TabBarController/MainTabBarController.swift:53]: if i don't write this code on device with touch id appeared this grey view on the navigationBar 
Simulator Screen Shot - iPhone 6s - 2023-02-05 at 15 49 54

**Review by AliaksandrNiehaj — APPROVED** [review]

**Review by Kengii — APPROVED** [review]

**Pull request merged**